src/main/kotlin/com/neva/gradle/fork/config/Config.kt

```diff
@@ -14,13 +16,18 @@ import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.util.*
 
-abstract class Config(val project: Project, val name: String) {
+abstract class Config(private val forkExtension: ForkExtension, val name: String) {
```

Comment:
just `extension` or `ext`. you will not use any other extensions so that fully qualified `forkExtension` is over-expressive for me (whole plugin is `fork` scoped :))